src/elements/common/back-button/BackButton.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @flow
+ * @file Back Button component
+ * @author Box
+ */
+
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { withRouter } from 'react-router-dom';
+import type { RouterHistory } from 'react-router-dom';
+import IconNavigateLeft from '../../../icons/general/IconNavigateLeft';
+import messages from '../messages';
+import PlainButton from '../../../components/plain-button';
+import './BackButton.scss';
+
+type Props = {
+    history: RouterHistory,
+};
+
+const BackButton = ({ history }: Props) => (
```

Comment:
Since EUA uses react-router this could be useful for other consumers? Maybe not since it uses our `withRouter`.

Reply:
Possibly. If that demand materializes, we can look at promoting this outside of elements to a more common location. The `withRouter` HOC should work as-is in EUA without issue.